client/my-sites/plugins/manage-site-plugins-dialog/index.jsx

```diff
@@ -21,8 +19,10 @@ export const ManageSitePluginsDialog = ( { isVisible, onClose, plugin } ) => {
 	const sitesWithPlugin = useSelector( ( state ) =>
 		getSiteObjectsWithPlugin( state, siteIds, plugin.slug )
 	);
-	const sitesWithoutPlugin = useSelector( ( state ) =>
-		getSiteObjectsWithoutPlugin( state, siteIds, plugin.slug )
+
+	const sites = useSelector( getSelectedOrAllSites );
```

Comment:
May I suggest sorting the list of sites that appear so all Atomic sites appear before Free sites? This is a bit of a nuisance because there's no Search for this dialog, and a user can have a lot of sites, whereas I think most users would be looking first for their Atomic sites. 

Just an idea I had when testing - feel free to ignore if you think it doesn't make sense. :)

Reply:
Good idea, I have added it here https://github.com/Automattic/wp-calypso/pull/88696/commits/c306c2cf9767b0ef1d29f8e06fc90b08f3e42699